bootstrap/src/routes/Welcome.js

```diff
@@ -161,6 +161,32 @@ const Welcome = () => {
                 .
             </p>
 
+            <h3>Add multi-tenant functionality and optional project picker</h3>
+            <ul>
+                <li>
+                    In <Code>Home.js</Code> replace all hard-coded project ID's with the one provided by
+                    <Code>useProjectId</Code> hook:{" "}
+                    <Code> {`<Headline projectId={projectId} … > … </Headline>`}</Code>
+                </li>
+                <li>
+                    <Code>ProjectId</Code> context object in <Code>/src/contexts/ProjectId.js</Code> stores
+                    actual project ID and provides it to the rest of the app. It also stores it in URL
+                    querystring so that the app can be easily embedded or linked with particular project
+                    pre-selected. If no project ID is found in URL then <Code>projectId</Code> from{" "}
+                    <Code>/src/constants.js</Code> is used as a default value.
+                </li>
+                <li>
+                    <Code>ProjectList</Code> context object in <Code>/src/contexts/ProjectList.js</Code>{" "}
```

Comment:
The <Code>ProjectList</Code> context object in <Code>/src/contexts/ProjectList.js</Code>{" "}